app/test_split_integer.py

```diff
@@ -2,20 +2,31 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    assert (
+        sum(split_integer(32, 6)) == 32
+    ), "5 + 5 + 5 + 5 + 6 + 6 should be equal to 32"
 
 
 def test_should_split_into_equal_parts_when_value_divisible_by_parts() -> None:
-    pass
+    assert (
+        split_integer(25, 5) == [5, 5, 5, 5, 5]
+    ), "25 is divisible by 5 so it should return [5, 5, 5, 5, 5]"
 
 
 def test_should_return_part_equals_to_value_when_split_into_one_part() -> None:
-    pass
+    assert (
+        split_integer(10, 1) == [10]
+    ), ("'numbers_of_parts' = 1, so func should return "
+        "list with one element equal to 'value'")
```

Comment:
If the number of parts is 1, the resulting list should contain the original value